test/e2e/traces_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build integration
```

Comment:
the tags again =/

Reply:
I will fix.  **It's just a comment**.  Sure, it is a comment the Go team wants us to use, but it is only a comment.